Examples/Sources/ChatbotExample/Views/MainChatView.swift

```diff
@@ -0,0 +1,156 @@
+import SwiftCrossUI
+
+// MARK: - Main Chat View
+
+struct MainChatView: View {
+    var viewModel: ChatbotViewModel
+    
+    var body: some View {
+        VStack(spacing: 0) {
+            // Header
+            ChatHeaderView(viewModel: viewModel)
+            
+            // Divider
+            Rectangle()
+                .fill(AppColors.border)
+                .frame(height: 1)
+            
+            // Content area
+            if viewModel.isThreadSelected {
+                ChatContentView(viewModel: viewModel)
+            } else {
+                EmptyThreadView(viewModel: viewModel)
+            }
+            
+            // Error message
+            if let errorMessage = viewModel.errorMessage {
+                ErrorMessageView(message: errorMessage)
+            }
+            
+            // Input area
+            if viewModel.isThreadSelected {
+                ChatInputView(
```

Comment:
Could probably just pass the input view the `viewModel` directly to simplify things, given that you're already doing so for the header (which needs less of the view model than this input view).